.github/workflows/publish_latest.yml

```diff
@@ -0,0 +1,162 @@
+name: Publish Latest
+
+on:
+  push:
+    branches: [master]
+  workflow_dispatch:
+    inputs:
+      force:
+        required: false
+        description: 'Set --force to force push this release'
+        default: ''
+      verbose:
+        description: "Set --verbose to get verbose build output"
+        required: false
+        default: ''
+
+env:
+  VERBOSE: ${{ github.events.input.verbose }}
+  FORCE_RELEASE: ${{ github.events.inputs.force }}
+
+jobs:
+  # Bump the `dev` tag to this commit in order to overwrite the `dev` GH prerelease
+  bump_github_release:
+    name: Bump dev tag
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Bump dev tag
+        run: |
+          git tag -f dev
+          git push -f origin dev
+
+  # Upload the build artifacts to the `dev` GH release, overwriting old artifacts
+  github_release:
+    name: Publish to GitHub Releases (${{ matrix.artifact }}-${{ matrix.target }})
+    needs: bump_github_release
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        artifact: [fluvio, fluvio-run]
+        target: [x86_64-unknown-linux-musl, x86_64-apple-darwin]
+    steps:
+      - name: Login GH CLI
+        run: gh auth login --with-token < <(echo ${{ secrets.GITHUB_TOKEN }})
+      - name: Download artifact
+        run: gh run download -R nicholastmosher/fluvio -n "${{ matrix.artifact }}-${{ matrix.target }}-${{ github.sha }}"
+      - name: Rename artifact
+        run: mv ${{ matrix.artifact }} ${{ matrix.artifact }}-${{ matrix.target }}
+      - name: Publish artifact
+        run: gh release upload -R nicholastmosher/fluvio --clobber dev ./${{ matrix.artifact }}-${{ matrix.target }}
+
+  # Build a docker image using the latest `fluvio-run` build for linux-musl
+  docker:
+    name: Publish Docker Image
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        rust: [stable]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Login GH CLI
+        run: gh auth login --with-token < <(echo ${{ secrets.GITHUB_TOKEN }})
+      - name: Download artifact
+        run: gh run download -R nicholastmosher/fluvio -n "fluvio-run-x86_64-unknown-linux-musl-${{ github.sha }}"
+
+      - name: Login to Docker Hub
+        run: docker login --username=${{ secrets.DOCKER_USERNAME }} --password=${{ secrets.DOCKER_PASSWORD }}
+      - name: Publish latest development Fluvio Image
+        run: |
+          export TAG="$(cat VERSION)-${{ github.sha }}"
+          k8-util/docker/build.sh "${TAG}" "./fluvio-run"
```

Comment:
use `make latest_image`

Reply:
`make latest_image` is too specific for what needs to happen here. What it does is build the docker image from the hardcoded file `"./target/x86_64-unknown-linux-musl/$(BUILD_PROFILE)/fluvio-run"`, instead of what we need here, which is to use `./fluvio-run`. That is the reason I rewrote the `k8-util/docker/build.sh` script to take the filename as an argument.